extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/OperationType.scala

```diff
@@ -21,12 +21,15 @@ object OperationType extends Enumeration {
 
   type OperationType = Value
 
+  // format: off
```

Comment:
Add this since
<img width="951" alt="image" src="https://github.com/apache/kyuubi/assets/46485123/f29472be-a565-4b99-b2a9-494b0bb9defe">
According to https://scalameta.org/scalafmt/docs/known-issues.html add this change

Reply:
alright, let's put the https://scalameta.org/scalafmt/docs/known-issues.html into comment

Reply:
feel free to turn off linter check if necessary

Reply:
Done